sentinelhub/api/batch/statistical.py

```diff
@@ -179,7 +181,18 @@ def cancel_job(self, batch_request: BatchStatisticalRequestType) -> Json:
         :param batch_request: It could be a batch request object, a raw batch request payload or only a batch
             request ID.
         """
-        return self._call_job(batch_request, "cancel")
+        return self._call_job(batch_request, "stop")
```

Comment:
I know I suggested rerouting, but tbh since we have a deprecation warning, it might as well stay as it is :thinking: 
I'm reverting it as a suggestion so you don't have to do anything.
```suggestion
        return self._call_job(batch_request, "cancel")
```